package.json

```diff
@@ -37,12 +37,12 @@
   "author": "JP Richardson <jprichardson@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "at-least-node": "^1.0.0",
     "graceful-fs": "^4.2.0",
```

Comment:
Not target of this PR, but should we update `graceful-fs` (and maybe all the dev dependencies too?). Just saw that there is already v`4.2.6`. Could be a nice move during our major version bump

Reply:
The `^` will already pull in the latest version; I don't see the point of bumping it explicitly.

I'm fine with bumping dev dependencies, but no need to tie that to the major bump. Feel free to jump on that whenever you have time.

Reply:
Ah ya true my bad!